plugin/cpp_auto_include.vim

```diff
@@ -122,6 +122,8 @@ module CppAutoInclude
       VIM::lines.each_with_index do |l, i|
         if l =~ /^\s*#\s*include/
           includes << [l, i+1]
+        elsif l =~ /^\s*namespace/ and includes.length == 1
+          includes << ['', i]
```

Comment:
Just change the line number of the place holder:

```
includes[0][-1] = i
```

This will save some bytes if there are multiple `namespace`s